docs/chart-and-series-types/histogram-series.md

```diff
@@ -57,4 +57,4 @@ Use the following setting to set up a histogram chart with a column chart:
         }
     }
 
-<iframe width="100%" height="470" style="null" src=https://jsfiddle.net/mushigh/18b1vun9/5/embedded/result allow="fullscreen"></iframe>
+<iframe width="100%" height="470" style="null" src=https://www.highcharts.com/samples/embed/highcharts/demo/histogram-column allow="fullscreen"></iframe>
```

Comment:
Samples that are added to `{product}/demo/***` are the ones that appear on our website demo section. We already have a histogram demo and we haven't planned another.

Instead, we should move this sample to `highcharts/series-histogram/column`, for use only by reference from the docs.